framework/sandstone.cpp

```diff
@@ -3431,6 +3435,14 @@ int main(int argc, char **argv)
         }
     }
 
+    if (test_list_randomize) {
+        sApp->use_strict_runtime = true;
```

Comment:
This shouldn't be implied just because it randomises.

Reply:
Agreed - and now removed.